docs/_documentations/troubleshooting.md

```diff
@@ -458,6 +458,21 @@ Codewind displays an error. In VS Code, the error appears in the Codewind log:
 2. Remove and add the repository back into Codewind with **Manage Template Sources**.
 3. Rebind the project to Codewind.
 
+<!--
+Codewind version: 0.9.0
+Action/Topic: Appsody with Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind/issues/938
+-->
+## Using Appsody stacks images from private Docker registries
+
+**Local scenario**
+Codewind works with fully qualified Appsody stacks only. The `stack:` property in the project's `.appsody-config.yaml` file needs to be in the following format: `hostname[:port]/username/reponame[:tag]`. Also, before you work with the stack, on the local system, enter `docker login` to the private registry.
```

Comment:
For Codewind to work with Appsody stacks images on private Docker registries, the stack must fully qualify the image name in their templates' `.appsody-config.yaml` configurations.

^ After some thought, I feel this may be more accurate, as the user don't really have visibility on the `stack:` property until after they create a project, nor is this under their control because it is up to the stack maintainers to build the stack in such a way that it has fully qualified names.